release-notes/5.0/preview/5.0.0-preview.4.md

```diff
@@ -30,7 +30,7 @@ Your feedback is important and appreciated. We've created an issue at [dotnet/co
 
 ## Visual Studio Compatibility
 
-**Visual Studio compatibility:** .NET 5 Preview 3 requires Visual Studio 2019 16.6 Preview to take full advantage of all its features. .NET 5 Preview 3 will not work properly in earlier versions of Visual Studio. Visit [Visual Studio Preview](https://visualstudio.microsoft.com/vs/preview/) to learn about VS previews.
+**Visual Studio compatibility:** .NET 5 Preview 4 requires Visual Studio 2019 16.6 Preview to take full advantage of all its features. .NET 5 Preview 4 will not work properly in earlier versions of Visual Studio. Visit [Visual Studio Preview](https://visualstudio.microsoft.com/vs/preview/) to learn about VS previews.
```

Comment:
let's remove the word preview here too since it needs VS 16.6
```suggestion
**Visual Studio compatibility:** .NET 5 Preview 4 requires Visual Studio 2019 16.6 to take full advantage of all its features. .NET 5 Preview 4 won't work properly in earlier versions of Visual Studio. Visit [Visual Studio Preview](https://visualstudio.microsoft.com/vs/preview/) to learn about VS previews.
```